### PR TITLE
Fixes text2ratvar - Unicode broke it - Fixes 8523

### LINKED ISSUE #8523: text2ratvar isn't working properly
## Reproduction:

say anything in ratvarian/see thing that uses text2ratvar
it will be missing most spaces and half the characters
example is ratvar's summon monologue being BAALGFAFBAGFCGGFGEF which is gibberish

### PR DESCRIPTION
**Pull request by JamieD1**
### Intent of your Pull Request

Fixes Clockcult text2ratvar

#### Changelog

:cl:  
bugfix: fixes text2ratvar
/:cl:

fixes #8523

### CODE REVIEW

File: code/__HELPERS/text.dm [hunk 1]
```diff
@@ -163,15 +163,15 @@
...
+					continue
 
 				last_char_group = SYMBOLS_DETECTED
 
 			//Space
 			if(32)
 				if(last_char_group == NO_CHARS_DETECTED || last_char_group == SPACES_DETECTED) //suppress double-spaces and spaces at start of string
 					continue
-				
+
```
Comment (Bibby0110) [inline code/__HELPERS/text.dm:174]: space

File: code/__HELPERS/text.dm [hunk 2]
```diff
@@ -163,15 +163,15 @@
 			// ~   |   @  :  #  $  %  &  *  +
 			if(126,124,64,58,35,36,37,38,42,43)			//Other symbols that we'll allow (mainly for AI)
 				if(last_char_group == NO_CHARS_DETECTED || !allow_numbers)	//suppress at start of string
-					continue	
+					continue
```
Comment (Bibby0110) [inline code/__HELPERS/text.dm:166]: space

File: code/__HELPERS/text.dm [hunk 3]
```diff
@@ -149,7 +149,7 @@
 			if(48 to 57)			//Numbers
 				if(last_char_group == NO_CHARS_DETECTED || !allow_numbers) //suppress at start of string
 					continue
-				
+
```
Comment (Bibby0110) [inline code/__HELPERS/text.dm:152]: space

**Review by Bibby0110 — CHANGES_REQUESTED** [review]

File: code/__HELPERS/text.dm (hunk 3 above)
Comment (Bibby0110) [reply code/__HELPERS/text.dm:152]: tabs*

File: code/__HELPERS/text.dm (hunk 2 above)
Comment (Bibby0110) [reply code/__HELPERS/text.dm:166]: tabs*

File: code/__HELPERS/text.dm (hunk 1 above)
Comment (Bibby0110) [reply code/__HELPERS/text.dm:174]: tabs*

**Review by Hopekz — APPROVED** [review]

**Pull request merged**